website/docs/en/api/modules/module-variables.mdx

```diff
@@ -126,52 +240,371 @@ componentsContext.keys().forEach(fileName => {
 
 `import.meta.webpackContext()` streamlines the process of module importation especially when you have a lot of files to manage. When using it, please avoid matching unnecessary files, as this might lead to significantly increased build time and output size.
 
-### import.meta.webpackHot (webpack-specific)
+### import.meta.webpackHot (Rspack/Webpack-specific)
```

Comment:
How about removing `(Rspack/Webpack-specific)` from the title? It makes the title too long

Reply:
This is used to distinguish from the standard. Perhaps we can try to use some label components

Reply:
I will handle this in an other pr

Reply:
How about move to comment:

```md
### import.meta.webpackHot

> This is an Rspack/Webpack specific API.
```